pkg/cosign/tuf/client.go

```diff
@@ -528,16 +562,40 @@ func cachedTargetsDir(cacheRoot string) string {
 	return filepath.FromSlash(filepath.Join(cacheRoot, "targets"))
 }
 
+func syncLocalMeta(from client.LocalStore, to client.LocalStore) error {
+	// Otherwise populate the in-memory local store with data fetched from the cache.
```

Comment:
nit: This comment isn't accurate, correct? Cause this can sync to or from local/disk, depending on if it's read or write?

Reply:
Yes! Sorry this was meant to be in caller comment, not internal. Moved, and clarified this comment.